morph/Grid.h

```diff
@@ -749,6 +750,74 @@ namespace morph {
             return expr_resampled;
         }
 
+        /*!
+         * Returns all the indices of the grid with a given radius (radius argument) of a given (x,y) location (loc argument)
+         *
+         * \param loc (x,y) metric location of the center of the circle
+         * \param radius radius defining the circle 
+         * \param inds_in_radius A vector of indices within the circle - supplied as a reference
+         */
+        void indices_in_radius (const morph::vec<C,2> loc, 
+                                 const C radius,
+                                 morph::vvec<I>& inds_in_radius)
+        {
+            morph::vvec<I> inds_in_circle;
+            morph::vvec<I> inds_in_previous_circle;
+            std::set<I> seen;
+            
+            // Find first index (middle of circle)
+            I current_ind = this->index_lookup(loc);
+            inds_in_radius.push_back(current_ind);
+            seen.insert(current_ind);
+            inds_in_circle.push_back(current_ind);
+
+            while (inds_in_circle.size() != 0){
```

Comment:
To be completely "on-style" you could place a space after ')' and before '{' here